README.md

```diff
@@ -217,6 +217,48 @@ func main() {
 // ]
 ```
 
+## Output ordering
+
+Sheriff converts the input struct into a basic structure using `map[string]interface{}`. This means that the generated 
+JSON will not have the same ordering as the input struct. If you need to have a specific ordering then a custom 
+implementation of the `KVStoreFactory` can be passed as an option.
+```go
+package main
+
+import (
+	"github.com/liip/sheriff/v2"
+	orderedmap "github.com/wk8/go-ordered-map/v2"
+)
+
+type OrderedMap struct {
+	*orderedmap.OrderedMap[string, interface{}]
+}
```

Comment:
nit:

```suggestion
type OrderedMap *orderedmap.OrderedMap[string, interface{}]
```

Not sure if that works - haven't tried it.

Reply:
I don't think it will work. The `orderedmap.OrderedMap` has private fields - and doing it this way doesn't grant access to the methods. We need this to access the `.Newest()`/`.Prev()` methods on the ordered map

https://go.dev/ref/spec#Type_definitions
>  It does not inherit any methods bound to the given type...

As such I believe that embedding is the only way to access this

Reply:
ah. You're absolutely right. 👍 thanks.